### PR TITLE
Github action to build/test wildfly-s2i on PR

### PR DESCRIPTION
**Pull request by jfdenise**
Build and run WildFly s2i tests using wildfly-cekit-modules PR branch.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,38 +1,100 @@
...
+           which s2i
+           s2i version
+      - name: Build 
+        run: |
+          . ~/cekit${{ env.CEKIT_VERSION }}/bin/activate
+          pushd wildfly-builder-image
+          cekit build --overrides=wf-cekit-overrides.yaml docker
+          popd
+          pushd wildfly-runtime-image
+          cekit build --overrides=wf-cekit-overrides.yaml docker
```
Comment (luck3y) [inline .github/workflows/main.yml:77]: Where does wf-cekit-overrides.yaml come from, is that in the image repo?

Comment (jfdenise) [reply .github/workflows/main.yml:77]: Yes v2 branch: https://github.com/wildfly/wildfly-s2i/blob/v2/wildfly-builder-image/wf-cekit-overrides.yaml

Comment (luck3y) [reply .github/workflows/main.yml:77]: You could consider something like this, if its easier to manage it all from the same repo.

`cekit -v build --overrides-file %cekit.build.overrides.file% \
 --overrides "{\"modules\": {\"repositories\": [{\"name\":\"wildfly-cekit-modules\",\"git\":{\"url\":\"%github.repo.wildfly-cekit-modules%\",\"ref\":\"%github.repo.wildfly-cekit-modules.ref%\"}}]}}"
`

**Review by luck3y — APPROVED** [review]

File: .github/workflows/main.yml (hunk 1 above)
Comment (jfdenise) [reply .github/workflows/main.yml:77]: Great suggestion! I will integrate in this PR.

Comment (luck3y) [reply .github/workflows/main.yml:77]: So something more like:

`
cekit -v build --overrides "{\"modules\": {\"repositories\": [{\"name\":\"wildfly-cekit-modules\",\"path\":\"../../wildfly-cekit-modules\"}]}}"
`
(the original example was from CI)

**Pull request merged**